integration_test/tests.exs

```diff
@@ -13,3 +13,4 @@ Code.require_file "cases/queue_test.exs", __DIR__
 Code.require_file "cases/stream_test.exs", __DIR__
 Code.require_file "cases/transaction_execute_test.exs", __DIR__
 Code.require_file "cases/transaction_test.exs", __DIR__
+Code.require_file "cases/hibernate_test.exs", __DIR__
```

Comment:
Alphabetically please because once we had a test file that wasn't being tested and that makes it easier to locate ;)